styles.css

```diff
@@ -79,17 +79,15 @@
   background-position-y: calc(8%);
   background-position-x: right;
   height: 718px;
-  width: 100%;
+  width: 375px;
 }
 
 .my-name {
   width: 85%;
   height: 160px;
   font-size: 40px;
   font-weight: 700;
-  padding-top: 90px;
-  padding-left: 24px;
-  padding-right: 24px;
+  padding: 90px 24px 0 24px;
   font-family:
     "Lucida Sans",
     "Lucida Sans Regular",
```

Comment:
- [ ] Great job 💪. Kindly make sure you use the correct `font-family` in your site as seen in your chosen Figma design 🙏